lib/gracenote/query_builder.rb

```diff
@@ -0,0 +1,20 @@
+module Gracenote
+  module QueryBuilder
+    def build_query(cmd, lang: nil, auth: nil)
+      Nokogiri::XML::Builder.new do |xml|
+        xml.QUERIES {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)